### ISSUE TITLE
Modify default path for .env files when using react-scripts (CRA)?

### ISSUE THREAD

**Issue by nirpeled** [gsoft-inc/craco#414]
Seems like react-scripts (CRA) is using an hard-coded path for reading the .env files (reading from project's folder).

How can I use craco or a craco plugin to override it?

Thanks :)

**Comment by dilanx** [comment]
I don't believe craco supports this at the moment, but I'll look into adding this.
